src/react/ui-elements/Help.tsx

```diff
@@ -1,21 +1,37 @@
 import { Tooltip } from '@scality/core-ui';
-import React from 'react';
-import type { Node } from 'react';
+import * as Core from '@scality/core-ui';
+import { IconName } from '@scality/core-ui/dist/components/icon/Icon.component';
```

Comment:
With the latest release of core-ui `v0.47.0`, we have the IconHelp integrated.  
So let's remove this Help component in zenko-ui and use the one in core-ui ;)